polar-core/src/events.rs

```diff
@@ -88,4 +88,9 @@ pub enum QueryEvent {
         operator: Operator,
         args: TermList,
     },
+
+    NextExternal {
+        call_id: u64,
+        term: Term,
```

Comment:
The name `term` is very vague. Would be nice if it provided more context, e.g. `iterable_term` or something.

Reply:
Sure